mlir/include/Catalyst/IR/CatalystOps.td

```diff
@@ -114,11 +114,12 @@ def PythonCallOp: Catalyst_Op<"pycallback",
   }];
 
   let arguments = (ins
+        Variadic<AnyType>:$inputs,
```

Comment:
For my understanding: Are we not guaranteed her to have a pointer to a memref?

Reply:
Yes, that is true. I could constrain it a bit more. Thanks!

Reply:
https://github.com/PennyLaneAI/catalyst/pull/596/commits/3034a3aa36a6c58141bac2569db3358bfb4484ba